src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable no-nested-ternary */
+
+import "./Person.scss";
+
+export const Person = ({ person }) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+      {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+      <p className="Person__partner">
```

Comment:
Also, add a blank line between the blocks
```suggestion
      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
      
      {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
      
      <p className="Person__partner">
```